src/components/TeamSection/TeamInformation.ts

```diff
@@ -213,42 +213,42 @@ export const projectManagers: TeamMember[] = [
   {
     id: 0,
     name: "Aarsh Shah",
-    affiliation: "Project Manager",
+    affiliation: "CampusBuddy",
     image: aarsh_jpg,
     linkedin: "https://www.linkedin.com/in/aarsh-shah-0a84161a9/",
   },
   {
     id: 1,
     name: "Hilton Luu",
-    affiliation: "Project Manager",
+    affiliation: "Loyalty Rewards Platform for Small Businesses",
```

Comment:
This is more of a description than a name for their app, please reach out to Hilton to confirm the project title